cob_hardware_config/common/upload_robot.launch

```diff
@@ -1,13 +1,34 @@
 <?xml version="1.0"?>
 <launch>
 
-    <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
-    <arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="robot_description" default="robot_description"/>
+	<arg name="config_path" default="$(find cob_hardware_config)/$(arg robot)/config"/>
```

Comment:
you are right , I though that planning_context needed it https://github.com/ipa-nhg/cob_manipulation/blob/Fix/MoveitDepends/cob_moveit_config/launch/planning_context.xml#L5 but it use the same path , only the package name is different